app/core/simulation.ts

```diff
@@ -1142,6 +1142,14 @@ export default class Simulation extends Schema implements ISimulation {
       case Effect.DEEP_MINER:
         if (types.has(Synergy.GROUND)) {
           pokemon.effects.add(effect)
+          const synergyLevel =
+            [
+              Effect.TILLER,
+              Effect.DIGGER,
+              Effect.DRILLER,
+              Effect.DEEP_MINER
+            ].indexOf(effect) + 1
+          pokemon.effectsSet.add(new GrowGroundEffect(synergyLevel, effect))
```

Comment:
This array already exists in SynergyEffects[Synergy.GROUND] which I think makes more sense to use, but isn't super important.